pkg/pillar/zfs/zfs.go

```diff
@@ -21,10 +21,8 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-const volBlockSize = uint64(16 * 1024)
-
-// taken from mkimage-raw-efi/install kubevirt RESERVE_EVE_STORAGE_SIZEGB
-const reserveEveStorageSizeGb = uint64(20 * 1024 * 1024 * 1024)
+// VolBlockSize is the default dataset block size
```

Comment:
What's the unit?  Makes sense to state that in the comment.

Reply:
Done in latest commit